src/backend/gporca/libnaucrates/src/operators/CDXLTableDescr.cpp

```diff
@@ -86,6 +87,26 @@ CDXLTableDescr::MdName() const
 	return m_mdname;
 }
 
+
+// table alias accessors
+const CMDName *
+CDXLTableDescr::MdAlias() const
```

Comment:
why not put them into h-file like CTableDescriptor.h?

Reply:
To keep consistent with CDXLTableDescr::MdName() which is also has a separated declaration. Personally I'd prefer to have in inline in header too.